Lib/fortran/fundamental.swg

```diff
@@ -167,6 +209,9 @@ SWIGINTERN SwigArrayWrapper SwigArrayWrapper_uninitialized() {
 
   %typemap(bindc) CTYPE* = FORTRAN_INTRINSIC_TYPE*;
 
+  ////////////////////////////////////////////////////////////////////////////////////////////////
+  // 1D Arrays 
```

Comment:
Minor quibble, but can you use separators consistent with the other code in the SWIG library code?

Reply:
Noted, I will change that.